helm-charts/FATE-Serving/templates/serving-server-module.yaml

```diff
@@ -27,8 +27,8 @@ data:
     #serviceRoleName=serving
     # cache
     #remoteModelInferenceResultCacheSwitch=false
-    #cache.type=local
-    #model.cache.path=
+    cache.type={{ .Values.cacheType | default "local" }}
+    model.cache.path=/root/.fate
     # local cache
     #local.cache.maxsize=10000
```

Comment:
@asdfsx  `.Values.cacheType` may not pass the correct variable, should be changed to `.Values.servingServer.cacheType`.

Reply:
my mistake
I will update it soon